recipes/lame/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
```

Comment:
the first 15 lines and the last 2 lines have been lifted from the ImageMagick feedstock's `build.sh`

Reply:
Yeah the ImageMagick feedstocks is a bit outdated it seems then. These flags are included in our legacy and new toolchains. So all of these can be dropped in the feedstock. This includes the `if`/`elif` below as well.

Also the `.la` (libtool archive) file removal is handled by `conda-build` directly. The fixes for this below preceded that release and can be dropped too.

Hopefully this simplifies things for you a bit going forward. :)